src/frontend/web_application/src/components/ReplyForm/presenter.jsx

```diff
@@ -32,6 +33,8 @@ class ReplyForm extends Component {
     super(props);
     this.state = {
       draftMessage: {},
+      isActive: true,
+      protocol: 'email',
```

Comment:
actually the user don't choose the protocol, it is the contact's available protocols witch determine what protocols will be used. Besides the user should be able to select the identity he wants to use.

Well, we don't have all of those things, so it is just fine like this :).